.github/workflows/build-and-deploy.yml

```diff
@@ -88,6 +107,22 @@ jobs:
           build-args: |
             SHA=${{ env.IMAGE_TAG }}
 
+      - name: Push ${{ env.DOCKER_IMAGE }} images for review
```

Comment:
As discussed, it is not necessary to separate the step for review apps

Reply:
fixed